src/recorder/recorder_worker_hdf5.cc

```diff
@@ -0,0 +1,343 @@
+
+  // Size of FFT
+  //hdf5_->WriteAttribute("FFT_SIZE", cfg_->OfdmCaNum());
+
+  // Length of cyclic prefix
+  hdf5_->WriteAttribute("CP_LEN", cfg_->CpLen());
+
+  hdf5_->WriteAttribute("OFDM_DATA_NUM", cfg_->OfdmDataNum());
+  hdf5_->WriteAttribute("OFDM_DATA_START", cfg_->OfdmDataStart());
+  hdf5_->WriteAttribute("OFDM_DATA_STOP", cfg_->OfdmDataStop());
+  hdf5_->WriteAttribute("OFDM_PILOT_SPACING", cfg_->OfdmPilotSpacing());
+  hdf5_->WriteAttribute("OFDM_CA_NUM", cfg_->OfdmCaNum());
+
+  // Beacon sequence type (string)
+  //hdf5_->WriteAttribute("BEACON_SEQ_TYPE", cfg_->beacon_seq());
+
+  // Pilot sequence type (string)
+  //hdf5_->WriteAttribute("PILOT_SEQ_TYPE", cfg_->pilot_seq());
+
+  // ******* Base Station ******** //
+  // Hub ID
+  hdf5_->WriteAttribute("BS_HUB_ID", cfg_->HubId());
+
+  // BS SDR IDs
+  hdf5_->WriteAttribute("BS_SDR_ID", cfg_->RadioId());
+
+  // Number of Base Station Cells
+  hdf5_->WriteAttribute("BS_NUM_CELLS", cfg_->CellId().size());
+
+  // How many RF channels per Iris board are enabled ("single" or "dual")
+  hdf5_->WriteAttribute("BS_CH_PER_RADIO", cfg_->NumChannels());
+
+  // Frame schedule (vec of strings for now, this should change to matrix when we go to multi-cell)
+  hdf5_->WriteAttribute("BS_FRAME_SCHED", cfg_->Frame().FrameIdentifier());
+
+  // RX Gain RF channel A
+  hdf5_->WriteAttribute("BS_RX_GAIN_A", cfg_->RxGainA());
+
+  // TX Gain RF channel A
+  hdf5_->WriteAttribute("BS_TX_GAIN_A", cfg_->TxGainA());
+
+  // RX Gain RF channel B
+  hdf5_->WriteAttribute("BS_RX_GAIN_B", cfg_->RxGainB());
+
+  // TX Gain RF channel B
+  hdf5_->WriteAttribute("BS_TX_GAIN_B", cfg_->TxGainB());
+
+  // Beamsweep (true or false)
+  hdf5_->WriteAttribute("BS_BEAMSWEEP", cfg_->Beamsweep() ? 1 : 0);
+
+  // Beacon Antenna
+  hdf5_->WriteAttribute("BS_BEACON_ANT", cfg_->BeaconAnt());
+
+  //If the antennas are non consective this will be an issue.
+  //hdf5_->WriteAttribute("ANT_OFFSET", antenna_offset_);
+  //hdf5_->WriteAttribute("ANT_NUM", num_antennas_);
+  //hdf5_->WriteAttribute("ANT_TOTAL", cfg_->getTotNumAntennas());
+
+  // Number of symbols in a frame
+  hdf5_->WriteAttribute("BS_FRAME_LEN", cfg_->Frame().NumTotalSyms());
+
+  // Number of uplink symbols per frame
+  hdf5_->WriteAttribute("UL_SLOTS", cfg_->Frame().NumULSyms());
+  hdf5_->WriteAttribute("DL_SLOTS", cfg_->Frame().NumDLSyms());
+  hdf5_->WriteAttribute("PILOT_SLOTS", cfg_->Frame().NumPilotSyms());
+  //PILOT_NUM
+  hdf5_->WriteAttribute("DL_CAL_SLOTS", cfg_->Frame().NumDLCalSyms());
+  hdf5_->WriteAttribute("UL_CAL_SLOTS", cfg_->Frame().NumULCalSyms());
+  hdf5_->WriteAttribute("UL_PILOT_SLOTS", cfg_->Frame().ClientUlPilotSymbols());
+  hdf5_->WriteAttribute("DL_PILOT_SLOTS", cfg_->Frame().ClientDlPilotSymbols());
+
+  // Reciprocal Calibration Mode
+  hdf5_->WriteAttribute("RECIPROCAL_CALIB",
+                        cfg_->Frame().IsRecCalEnabled() ? 1 : 0);
+
+  // ******* Clients ******** //
```

Comment:
Remove commented code?
Also, it may be too much of a change, but it might be better to just include the JSON files as metadata instead of individually recording each parameter. I thought about it for Sounder, but that might make issues with older datasets.